benchmarks/docker/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+# See dockerhub for different versions of kafka and zookeeper
+# https://hub.docker.com/r/wurstmeister/kafka/
+# https://hub.docker.com/r/wurstmeister/zookeeper/
+version: '2'
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper:3.4.6
+    ports:
+      - "2181:2181"
+  kafka:
+    image: wurstmeister/kafka:1.1.0
+    ports:
+      - "9092:9092"
+    environment:
+      # On Mac you may need to change `localhost` in KAFKA_ADVERTISED_LISTENERS to the ip/host of your docker machine ip
+      KAFKA_ADVERTISED_LISTENERS: INSIDE://localhost:9092,OUTSIDE://localhost:9094
```

Comment:
Nice. I was playing around with Kafka in docker containers as well and noticed that the `KAFKA_ADVERTISED_HOST_NAME` is deprecated.